packages/ia-activity-indicator/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@internetarchive/ia-activity-indicator",
+  "version": "0.0.0",
+  "description": "A customizable activity indicator.",
+  "license": "AGPL-3.0-only",
+  "main": "index.js",
+  "module": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "lib/**/*",
+    "!lib/demo",
+    "src/**/*",
+    "index.d.ts",
+    "index.js"
+  ],
+  "scripts": {
+    "prepare": "yarn run build",
+    "build": "tsc -p tsconfig.build.json",
+    "start": "concurrently \"tsc --watch\" \"es-dev-server --app-index demo/index.html --node-resolve --open --watch --preserve-symlinks\"",
+    "start:compatibility": "concurrently \"tsc --watch\" \"es-dev-server --app-index demo/index.html --node-resolve --open --watch --compatibility all --preserve-symlinks\"",
+    "lint:eslint": "eslint --ext .ts . --ignore-path .gitignore",
```

Comment:
Looks like there's a lot more TypeScript-specific code in here that can be removed